.nvmrc

```
@@ -0,0 +1 @@
+12.7.0
```

This should be enough:
```suggestion
12
```